format/Flight.proto

```diff
@@ -246,20 +246,11 @@ message FlightEndpoint {
   Ticket ticket = 1;
 
   /*
-   * A list of locations where this ticket can be redeemed. If the list is
+   * A list of URIs where this ticket can be redeemed. If the list is
    * empty, the expectation is that the ticket can only be redeemed on the
    * current service where the ticket was generated.
    */
-  repeated Location location = 2;
```

Comment:
I think it might be better to keep the location message and replace the internal with a single field URI.  This makes adding additional fields easier in the future if needed. Also, documenting supported protocols might be useful.

Reply:
Yeah, makes sense.